.env/backend.env

```diff
@@ -1,5 +1,6 @@
 NODE_ENV=development
 PORT=3000
+PASS=PassW0rd!
```

Comment:
⚠️ secrets shouldn't be commited

Reply:
Yes, you're right, but in this case it's for testing, not production. The password must be changed for production.